From 5be956455faf93b404ca0b4f2fbdb31c11c7187a Mon Sep 17 00:00:00 2001
From: Bernhard Mallinger <bernhard.mallinger@eox.at>
Date: Thu, 7 Jan 2021 12:33:28 +0100
Subject: [PATCH] Set async as default

---
 pygeoapi/api.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git pygeoapi/api.py pygeoapi/api.py
index 09280b2..0f06bec 100644
--- pygeoapi/api.py
+++ pygeoapi/api.py
@@ -2169,7 +2169,7 @@ tiles/{{{}}}/{{{}}}/{{{}}}/{{{}}}?f=mvt'
         headers_['Location'] = url
 
         outputs = status = None
-        is_async = data.get('mode', 'auto') == 'async'
+        is_async = data.get('mode', 'auto') != 'sync'
 
         if is_async:
             LOGGER.debug('Asynchronous request mode detected')
-- 
2.25.1

